db/migrate/20190128014059_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
```

Comment:
I generally prefer UUIDs for record IDs (for various reasons, which am happy to go into in chat) – if you're cool with that, let me know and I can add a commit to this PR with the needed changes.

Reply:
Yup sounds good to me